src/database/DataTypes/VaccineVialMonitorStatus.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Sustainable Solutions (NZ) Ltd. 2020
+ * mSupply Mobile
+ */
+
+import Realm from 'realm';
+
+export class VaccineVialMonitorStatus extends Realm.Object {}
```

Comment:
I mean it's a long name.. but.. I dunno..!

Reply:
...but not as long as `VaccineVialMonitorStatusLog`!